controllers/apply/index.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
+const { get, isEmpty, set, unset } = require('lodash');
+const { matchedData } = require('express-validator/filter');
+const { validationResult } = require('express-validator/check');
 const express = require('express');
+const moment = require('moment');
+const path = require('path');
+const Raven = require('raven');
 
 const appData = require('../../modules/appData');
-const { createFormRouter } = require('../helpers/create-form-router');
+const cached = require('../../middleware/cached');
+
 const reachingCommunitiesForm = require('./reaching-communities/form-model');
 const digitalFundingDemoForm = require('./digital-funding-demo/form-model');
 
+// How many days to extend the user's session by
+const EXTENDED_SESSION_DURATION_IN_DAYS = 7;
```

Comment:
Possibly we can get rid of this as the session is now 7 days by default rather than a session cookie?